### DISCUSSION TITLE
Is the OKDerators catalog ready to use?

### DISCUSSION THREAD

**Discussion by rasystems1** [okd-project/okd#2053] [Q&A] [answered]
Hello team, I have recently found that a catalog of operators (OKDerators https://github.com/okd-project/okderators-catalog-index) is being worked on, among which is the LVM Storage operator, and I am interested in trying it out. My question is, can this catalog currently be used for production by the operators that are available? since I have installed the LVM Operator and it seems to work well, but I don't know if it can be used in production environments. Many thanks in advance for the responses.

**Comment by JaimeMagiera** [comment, marked as answer]
No, we have not signed off on the catalog operators being used in Production yet. That will happen when the first version of OKD SCOS is released.
